benchmark/main.cpp

```diff
@@ -207,15 +194,8 @@ do {                                                                           \
         CODE;                                                                  \
     }                                                                          \
                                                                                \
-    tsc = mbedtls_timing_hardclock();                                          \
-    for( j = 0; j < 1024; j++ )                                                \
-    {                                                                          \
-        CODE;                                                                  \
-    }                                                                          \
-                                                                               \
-    mbedtls_printf( "%9lu KB/s,  %9lu cycles/byte\r\n",                        \
-                     i * BUFSIZE / 1024,                                       \
-                     ( mbedtls_timing_hardclock() - tsc ) / ( j * BUFSIZE ) ); \
+    mbedtls_printf( "%9lu KB/s\r\n",                                           \
+                     i * BUFSIZE / 1024 );                                     \
```

Comment:
Since we are changing prints we should also update examples log file for CI to pass https://github.com/ARMmbed/mbed-os-example-tls/blob/master/tests/benchmark.log